frontend/src/components/FormFooter.jsx

```diff
@@ -0,0 +1,61 @@
+import React from "react";
+export default function FormFooter({
+  isFilePicked,
+  contentFile,
+  length,
+  setFile,
+  handleSubmit,
+}) {
+  function getNiceName(requiredLength) {
```

Comment:
Please add  `// Method` or `// Methods` above the function

Reply:
Leave a row above //Method